doc/meetings/2022-01-11.md

```diff
@@ -0,0 +1,37 @@
+# Node.js  Loaders Team Meeting 2022-01-11
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=js_NQp-bomM
+* **GitHub Issue**: https://github.com/nodejs/loaders/issues/58
+
+## Present
+
+* Geoffrey Booth: @GeoffreyBooth
+* Bradley Farias: @bmeck
+* Jacob Smith: @jakobjingleheimer
+* Maël Nison: @arcanis
+
+## Agenda
+
+## Announcements
+
+*Extracted from **loaders-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
```

Comment:
```
*Extracted from **loaders-agenda**
```

Something about this is wonky—is this supposed to be a bullet point or italics?

Reply:
I don’t know, that’s just part of the tempalte.